Makefile

```diff
@@ -213,9 +213,13 @@ clean-temporary: ## Remove all temporary files and folders.
 	rm -f *.kubeconfig
 
 .PHONY: clean-release
-clean-release: ## Remove the release folder.
+clean-release: clean-release-git ## Remove the release folder.
 	rm -rf $(RELEASE_DIR)
 
+.PHONY: clean-release-git
+clean-release-git: ## Restores the git files usually modified during a release
```

Comment:
Woah thanks for adding this! I'm assuming this partially solves #2681? I know that it was attempted to try not to modify `manager_image_patch.yaml` and `manager_pull_policy.yaml` during releases/e2e tests, but this seems like a good solution around that. Great find 🚀